bdb/rep.c

```diff
@@ -69,6 +69,7 @@ int gbl_dump_sql_on_repwait_sec = 10;
 
 extern struct thdpool *gbl_udppfault_thdpool;
 extern int gbl_commit_delay_trace;
+extern int gbl_debug_txn_sleep;
```

Comment:
A better place for this debug tunable is the [debug swtiches](https://github.com/bloomberg/comdb2/blob/master/util/debug_switches.c#L27).

Reply:
It seems like these switches only accept 0/1 values. This wouldn't allow the user to use the tunable to [specify the sleep period.](https://github.com/bloomberg/comdb2/pull/3547#pullrequestreview-1186596273)